specification/sigma-appendix-custom-taxonomy.md

```diff
@@ -0,0 +1,96 @@
+# Custom Taxonomy
+
+The following document defines the field names and log sources that use custom taxonomy
+
+- Version 1.0.0
+- Release date 2025-07-30
+
+<!-- mdformat-toc start --slug=github --no-anchors --maxlevel=6 --minlevel=2 -->
+
+- [Warning](#warning)
+- [Network Events](#network-events)
+  - [Network Connection](#network-connection)
+  - [Network DNS](#network-dns)
+- [History](#history)
+
+<!-- mdformat-toc end -->
+
+## Warning
+
+You MUST use the `taxonomy` field in the rule otherwise it is considered as `sigma`.
+
+## Network Events
+
+Network events can be defined with the generic logsource category *network*.
+
+The event scope can be further restricted with *service*.
+
+### Network Connection
+
+This logsource covers networks connection in general.
+
+```yml
+category: network
+service: connection
+```
+
+The field names follow the field names used in [Elastic Common Schema](https://www.elastic.co/guide/en/ecs/current/ecs-network.html) events `taxonomy: ecs`:
+
+| Field Name           | Example Value                  | Comment                                           |
+| -------------------- | ------------------------------ | ------------------------------------------------- |
+| @timestamp           | 2025-05-08T14:38:35.000Z       | Event timestamp in UTC                            |
+| event.duration       | 0.005726                       | Connection duration in seconds                    |
+| event.id             | CXWfMc4eWKNBm1O4fl             | Unique connection identifier                      |
+| network.type         | ipv4                           | Network layer type (e.g., ipv4, ipv6, ipsec)      |
+| network.transport    | tcp                            | Transport layer protocol (e.g., tcp, udp)         |
+| network.protocol     | http                           | Application layer protocol (e.g., http, dns, ssh) |
```

Comment:
We should ensure that the most common values are specified here or define a rule for creating these values, e.g. "lower case letters, spaces replaces with underscores".